bgpd/bgp_route.c

```diff
@@ -10489,9 +10492,14 @@ DEFUN (show_ip_bgp,
 		return bgp_show_filter_list(vty, bgp, argv[idx + 1]->arg, afi,
 					    safi, bgp_show_type_filter_list);
 
-	if (argv_find(argv, argc, "statistics", &idx))
-		return bgp_table_stats(vty, bgp, afi, safi);
-
+	if (argv_find(argv, argc, "statistics", &idx)) {
+		if (uj)
```

Comment:
Why is this function allocating the json object and then freeing it?  It doesn't use it right?

Reply:
this can be for further usage ( as the calling function calls some vty commands that are not json compatible yet... because it is not scope for now, all json stuff will be internal. ok